documentation/asciidoc/topics/yaml/datagrid_service.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  # Names the cluster.
+  name: {example_crd_name}
+spec:
+  # Specifies the number of nodes in the cluster.
+  replicas: 6
+  service:
+    # Configures the service type as Data Grid.
+    type: DataGrid
+    # Configures storage resources.
+    container:
+      storage: 2Gi
+      ephemeralStorage: false
+      storageClassName: my-storage-class
+    # Configures cross-site replication.
+    sites:
+      local:
+      name: azure
+      expose:
+        type: LoadBalancer
+      # Configures backup locations.
+      locations:
+      - name: azure
+        url: openshift://api.azure.host:6443
+        secretName: azure-token
+      - name: aws
+        clusterName: {example_crd_name}
```

Comment:
`clusterName` and `namespace` options aren't available in 2.0.x yet

Reply:
yeah I'll have to make sure this doesn't get included when we backport to 2.0.x